build/ci/evergreen_common.yml

```diff
@@ -52,12 +52,21 @@ functions:
         <<: *go_options
         working_dir: src/github.com/mongodb/mongodb-atlas-cli/bin
         shell: bash
+        env:
+          <<: *go_env
         script: |
           set -Eeou pipefail
-          mkdir -p mongodb-database-tools
-          curl -sfL https://fastdl.mongodb.org/tools/db/mongodb-database-tools-rhel70-x86_64-100.10.0.tgz | tar zx -C mongodb-database-tools --strip-components 1
-          mv mongodb-database-tools/bin/* .
-          rm -rf mongodb-database-tools
+          if [[ "$(uname -s)" == "Darwin"* ]]; then
+            curl -sfLO "https://fastdl.mongodb.org/tools/db/mongodb-database-tools-macos-arm64-100.10.0.zip"
+            unzip mongodb-database-tools-macos-arm64-100.10.0.zip
+            mv mongodb-database-tools-macos-arm64-100.10.0/bin/* .
+            rm -rf mongodb-database-tools-macos-arm64-100.10.0 mongodb-database-tools-macos-arm64-100.10.0.zip
+          else
+            mkdir -p mongodb-database-tools
+            curl -sfL "https://fastdl.mongodb.org/tools/db/mongodb-database-tools-rhel70-x86_64-100.10.0.tgz" | tar zx -C mongodb-database-tools --strip-components 1
```

Comment:
please use a `${dbtools_version}` env var (set to 100.10.0)

Reply:
done